tap_jira/streams.py

```diff
@@ -134,6 +135,12 @@ def sync_on_prem(self):
                 path = "/rest/api/2/project/{}/version".format(project["id"])
                 pager = Paginator(Context.client, order_by="sequence")
                 for page in pager.pages(VERSIONS.tap_stream_id, "GET", path):
+                    for each_page in page:
+                        # Transform userReleaseDate and userReleaseDate to 'yyyy-mm-dd' format.
```

Comment:
Ad details like why this additional transformation is required for these two fields.

Reply:
Added detail comment.